src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -1241,6 +1237,10 @@ export default {
 				}
 			});
 
+			if (this.$route.hash === '#show-basket') {
+				this.showBasket();
+			}
+
```

Comment:
This may be better placed within the `then` block of the `validateBasket` call below, I would suggest after `this.balanceLeftoverCredits();` on line 1268 as the auth status could be useful for the showBasket call as well. This way we'd have scaffolded all the basket data/user state and know we've validated the basket before potentially showing it.

Reply:
ok done.